core/sail/fts/elasticsearch/src/main/java/org/eclipse/rdf4j/sail/elasticsearch/ElasticsearchIndex.java

```diff
@@ -173,7 +174,7 @@ public void initialize(Properties parameters)
 		throws Exception
 	{
 		super.initialize(parameters);
-		indexName = parameters.getProperty(INDEX_NAME_KEY, DEFAULT_INDEX_NAME);
+		indexName = parameters.getProperty(INDEX_NAME_KEY, UUID.randomUUID().toString());
```

Comment:
I was suggesting tests would be better isolated if they relied on distinct cluster names rather than index names. Users are expected to choose index/cluster config as appropriate, defaults are set to give a single distributed index, which is the closest match to the defaults of the other search impls. A user can run up multiple sail servers and they would all use the same index.
Having random indices doesnt give a config that is distributed out-the-box  - you could achieve something similar with just a bunch of lucene sails writing to different directories. Basically, I'm following the principal that the default for a distributed search sail should be something that is distributed.